src/shared/apis/index.ts

```
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+import { preventMultipleRefreshToken } from './preventMultipleRefreshToken';
```

ESLint

[error] Dependency cycle detected.

(import/no-cycle)